### PR TITLE
*: remove TiFlash http service

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
### What problem does this PR solve? 

remove TiFlash http service

### What is changed and how it works?

remove `http_port` configuration

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: components/playground/instance/tiflash_config.go [hunk 1]
```diff
@@ -106,12 +106,19 @@ func writeTiFlashConfig(w io.Writer, version utils.Version, tcpPort, httpPort, s
 	logDir := fmt.Sprintf("%s/log", deployDir)
 	ip := AdvertiseHost(host)
 	var conf string
+
+	port := "#"
+	// For 7.1.0 or later, TiFlash HTTP service is removed, so we don't need to set http_port
+	if !tidbver.TiFlashNotNeedHTTPPortConfig(version.String()) {
```
Comment (kaaaaaaang) [inline components/playground/instance/tiflash_config.go:112]: I heard that TiFlash donot need whole config file？

Comment (Lloyd-Pottiger) [reply components/playground/instance/tiflash_config.go:112]: just replace the config file with args, still need some configuration.

File: components/playground/instance/tiflash_config.go [hunk 2]
```diff
@@ -106,12 +106,19 @@ func writeTiFlashConfigForStartViaConfig(w io.Writer, version utils.Version, tcp
 	logDir := fmt.Sprintf("%s/log", deployDir)
 	ip := AdvertiseHost(host)
 	var conf string
+
+	port := "#"
+	// For 7.1.0 or later, TiFlash HTTP service is removed, so we don't need to set http_port
```
[the remark this answers is not in the document]
Comment (Lloyd-Pottiger) [reply components/playground/instance/tiflash_config.go]: removed.

**Review by kaaaaaaang — APPROVED** [review]

**Pull request merged**